explorer/client/src/utils/stringUtils.ts

```diff
@@ -24,15 +20,7 @@ export const handleCoinType = (str: string): string =>
         ? 'SUI'
         : str.match(/^([a-zA-Z0-9:]*)<([a-zA-Z0-9:]*)>$/)?.[2] || str;
 
-export const processDisplayValue = (display: { bytes: number[] } | string) => {
-    const url =
-        typeof display === 'object' && 'bytes' in display
-            ? asciiFromNumberBytes(display.bytes)
-            : display;
-    return typeof url === 'string' ? transformURL(url) : url;
-};
-
-function transformURL(url: string) {
+export function transformURL(url: string) {
```

Comment:
This function (`transformURL`) maps the IPFS identifier into a searchable URL that uses the free gateway service (`https://ipfs.io/ipfs/`). The problem with this is that IPFS's free gateway service can be slow. See the Aardvark example available on the IPFS website: https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/Aardvark.html

Hence, there is a need to see how the UX works when loading a slow IPFS resource. Could an example be added to the mock dataset with an IPFS reference and a video be taken to gain visibility into this?

Such a video would also provide evidence that the handling of IPFS resources is working as expected.

Reply:
@Andrew47 i didn't change IPFS handling works. This sounds like an issue for another PR.